corgi/core/models.py

```diff
@@ -302,6 +306,14 @@ def save_product_taxonomy(self):
 
         return None
 
+    def save(self, *args, **kwargs):
+        if "completion_time" in self.meta_attr:
+            dt = datetime.datetime.strptime(
+                self.meta_attr["completion_time"].split(".")[0], "%Y-%m-%d %H:%M:%S"
+            )
+            self.completion_time = make_aware(dt, timezone=TZINFO)
```

Comment:
If we're making completion_time a proper field, let's set it in the collector itself, just like we set e.g. name, tags, or type.

You can also parse the date with Django's utils:

```python
from django.utils import timezone, dateparse
make_aware(dateparse.parse_datetime(completion_time))
```

`make_aware` uses the timezone that is already configured in settings, which is UTC. Re-defining it in an extra TZINFO constant is not necessary.

Reply:
thx will do!